pkgs/development/tools/misc/libtool/libtool2.nix

```diff
@@ -26,9 +26,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  # Don't run the native `strip' when cross-compiling.  This breaks at least
-  # with `.a' files for MinGW.
-  dontStrip = stdenv.hostPlatform != stdenv.buildPlatform;
+  dontStrip = false;
```

Comment:
Why not remove this?

Reply:
originally it was to avoid a mass rebuild, but given this is a mass rebuild anyway, removing it makes sense.